js/stores/windowStore.js

```diff
@@ -752,21 +750,18 @@ const doAction = (action) => {
         }
         break
       }
-    case windowConstants.WINDOW_ON_WINDOW_UPDATE:
     case appConstants.APP_WINDOW_READY:
-      {
-        const oldInfo = windowState.get('windowInfo', Immutable.Map())
-        let windowValue = makeImmutable(action.windowValue)
-
-        if (windowValue.get('focused')) {
-          windowValue = windowValue.set('focusTime', performance.timing.navigationStart + performance.now())
```

Comment:
This should be set whenever window is updated (not only when it's ready) as we want to save which window **last** had focus. Also we can use `new Date().getTime()` as milliseconds are perfectly fine, we don't need to resolve to a microsecond for this.